services/frontend-v3/src/components/userTable/UserTableView.jsx

```diff
@@ -217,82 +290,66 @@ class UserTableView extends Component {
+        ...getColumnSearchProps(
           "formatCreatedAt",
-          this.props.intl.formatMessage({
+          props.intl.formatMessage({
             id: "admin.registered",
-            defaultMessage: "Registered"
+            defaultMessage: "Registered",
           })
-        )
+        ),
       },
       {
-        title: this.props.intl.formatMessage({
+        title: props.intl.formatMessage({
           id: "admin.tenure",
-          defaultMessage: "Tenure"
+          defaultMessage: "Tenure",
         }),
         dataIndex: tenureState,
         key: "tenure",
         sorter: (a, b) => {
           return a[tenureState].localeCompare(b[tenureState]);
         },
-        ...this.getColumnSearchProps(
+        ...getColumnSearchProps(
           tenureState,
-          this.props.intl.formatMessage({
+          props.intl.formatMessage({
             id: "admin.tenure",
-            defaultMessage: "Tenure"
+            defaultMessage: "Tenure",
           })
-        )
+        ),
       },
       {
-        title: this.props.intl.formatMessage({
+        title: props.intl.formatMessage({
           id: "admin.profileStatus",
-          defaultMessage: "Profile Status"
+          defaultMessage: "Profile Status",
         }),
-        render: record => {
-          return this.renderStatusDropdown(
+        render: (record) => {
+          return renderStatusDropdown(
             record.id,
             record.user.inactive,
             record.flagged
           );
-        }
-      }
+        },
+      },
     ];
 
-    return user_table_columns;
+    return tableColumns;
   };
 
-  render() {
-    const { data, size, statuses, handleApply } = this.props;
-    return (
-      <>
-        <PageHeader
-          title={this.props.intl.formatMessage({
-            id: "admin.user.table",
-            defaultMessage: "Users Table"
-          })}
-          extra={[
-            <Button
-              type="primary"
-              icon={<CheckCircleOutlined />}
-              size={size}
-              onClick={handleApply}
-              disabled={Object.entries(statuses).length === 0}
-            >
-              {this.props.intl.formatMessage({
-                id: "admin.apply",
-                defaultMessage: "Apply"
-              })}
-            </Button>
-          ]}
-        />
-        <Row gutter={[0, 8]}>
-          <Col span={24}>
-            <Table columns={this.tableColumns()} dataSource={data} />
-          </Col>
-        </Row>
-      </>
-    );
-  }
+  return (
+    <>
+      <PageHeader
+        title={props.intl.formatMessage({
+          id: "admin.user.table",
+          defaultMessage: "Users Table",
+        })}
+        extra={[applyButton()]}
+      />
+      <Row gutter={[0, 8]}>
+        <Col span={24}>
+          <Table columns={userTableColumns()} dataSource={data} />
+        </Col>
+      </Row>
+    </>
+  );
 }
 
 export default injectIntl(UserTableView);
```

Comment:
avoid using this as much as possible by using <formatedMessage> component

Reply:
Talked with Rizvi about this, on the ant design website they were doing it like this and I was going to be using <div> but changed to this instead. The tables are different with their features to make it into a component.